site/en/docs/devtools/javascript/breakpoints/index.md

```diff
@@ -140,10 +194,10 @@ To set an XHR breakpoint:
     anywhere in an XHR's request URL.
 5.  Press Enter to confirm.
 
-{% Img src="image/admin/BQXfCgOqQp7usvLOSySx.png", alt="Creating an XHR breakpoint.", width="800", height="698" %}
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/AAL4i6FKWOaIEdreyLjE.png", alt="Creating an XHR breakpoint.", width="800", height="756" %}
```

Comment:
Not about this image but comments for the walkthrough above. This section title already mention "XHR/Fetch breakpoint", so i think we can 

1. remove the "extra notes"

```
{% Aside %}
**Note:** This feature also works with [Fetch][10] requests.
{% endAside %}
```

2. Replace  all wording of`XHR breakpoint` to `XHR/ fetch breakpoint`